src/main/resources/plugin.yml

```diff
@@ -1,5 +1,5 @@
 name: TradeShop
-version: 1.6.0-BETA
+version: 1.7.0-BETA
```

Comment:
The most appropriate tag would be ``-DEV``, but we never used it so...

Reply:
I can start using it, didnt know that you added it

Reply:
It’s always been there tbh. It’s purpose is to tell that it’s not supposed to be for public use.

Reply:
*production

Reply:
Ill start using it